src/main/java/ch/njol/skript/expressions/ExprSpawner.java

```diff
@@ -52,7 +52,7 @@
 	@Override
 	@Nullable
 	public EntityType convert(final Block b) {
-		if (b.getType() != Material.SPAWNER)
+		if (b.getType() != (Skript.isRunningMinecraft(1, 13) ? Material.SPAWNER : Material.LEGACY_MOB_SPAWNER))
```

Comment:
Its been a while since I touched item stuff, but I think you can just do
```java
Aliases.javaItemType("spawner").getMaterial()
```
this will return the correct Material per version, without having to use Bukkit types

Reply:
Would the latest commit work? xD

Reply:
It looks like it should work, have you tested it on at least 1.13 (or 1.14) and 1.12.2?

Reply:
Since spawner has had an API since a long time, I'd expect it to work with 1.9.4, though, if it happens to be too much of an issue. Don't bother.